subscription/create_my_subscription_data.go

```diff
@@ -0,0 +1,31 @@
+// Package subscription implements the Subscription NEX protocol
+package subscription
+
+import (
+	"fmt"
+
+	nex "github.com/PretendoNetwork/nex-go"
+	"github.com/PretendoNetwork/nex-protocols-go/globals"
+)
+
+// CreateMySubscriptionData sets the CreateMySubscriptionData handler function
+func (protocol *SubscriptionProtocol) CreateMySubscriptionData(handler func(err error, client *nex.Client, callID uint32, content []byte)) {
+	protocol.CreateMySubscriptionDataHandler = handler
+}
+
+func (protocol *SubscriptionProtocol) handleCreateMySubscriptionData(packet nex.PacketInterface) {
+	if protocol.CreateMySubscriptionDataHandler == nil {
+		fmt.Println("[Warning] SubscriptionProtocol::CreateMySubscriptionData not implemented")
+		go globals.RespondError(packet, SubscriptionProtocolID, nex.Errors.Core.NotImplemented)
+		return
+	}
+
+	client := packet.Sender()
+	request := packet.RMCRequest()
+
+	callID := request.CallID()
+	parameters := request.Parameters()
+	content := parameters[8:]
```

Comment:
Adding on to what @DaniElectra said, even if this is unknown it would be nice to still make it available in the handler. Since this is 8 bytes, even just treating it as a uint64 would suffice

Ideally all data sent in the request should be available in the handler